packages/deployer/src/cli.ts

```diff
@@ -67,6 +67,32 @@ async function onDeployCommandAsync(argv: CliOptions): Promise<void> {
     const deployerArgs = deployerArgsString.split(',');
     await commands.deployAsync(argv.contract as string, deployerArgs, deployerOpts);
 }
+/**
+ * Creates a set of contracts to compile.
+ * @param contractDirectoriesList Comma separated list of contract directories
+ * @return Set of contract directories
+ */
+function getContractDirectoriesFromList(contractDirectoriesList: string): Set<ContractDirectory> {
+    const directories = new Set();
+    const namespacedDirectories: string[] = contractDirectoriesList.split(",");
```

Comment:
You'll likely get build failures from tslint/prettier as it prefers single quotes for strings over double quotes, and some spacing around directives `if(` etc.

Our configuration file for prettier is:
```
{
    "tabWidth": 4,
    "printWidth": 120,
    "trailingComma": all,
    "singleQuote": true
}
```

Or you can `yarn run prettier` in the top level directory and it should fix up most for you!